### PR TITLE
Add console log colors

### PR DESCRIPTION
**Pull request by bradwade**
Make the CLI report a little prettier.
- Added red and green to error and passes. 
- Removed extra space. 
- Standardized on single quotes inside console.log.

### CODE REVIEW

File: gulp/tasks/compare.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ gulp.task('compare', function (done) {
 
   function storeFailedDiffImage(testPath, data) {
     var failedDiffFilename = getFailedDiffFilename(testPath);
-    console.log('Storing diff image in ', failedDiffFilename);
+    console.log('Storing diff image in', failedDiffFilename);
```
Comment (garris) [inline gulp/tasks/compare.js]: As long as we're at it, maybe the text should be shortened to: 
"       See: "

Essentially indent the file info to start with the error data value.

Comment (bradwade) [inline gulp/tasks/compare.js]: Good idea. Will update PR.

**Comment by bradwade** [comment]
I indented the label three spaces which is sufficient to show hierarchy, economizes space (I have some looong file names), lines up the colons, and places the file paths at the beginning of the error message.

**Comment by garris** [comment]
Looks good -- merged -- thanks!

**Pull request merged**